src/app/pages/test-report/cat-a-mod1/components/activity-code-4-modal/__tests__/activity-code-4-modal.spec.ts

```diff
@@ -54,4 +55,19 @@ describe('ActivityCode4Modal', () => {
     });
   });
 
+  describe('onCancel', () => {
+    it('should call dismiss with CANCEL', async () => {
+      spyOn(component['modalController'], 'dismiss');
```

Comment:
Can we not use this syntax, mentioned before to access private providers we use this pattern

Declare
`let modalController: ModalController;`

Add this to beforeEach setup

`modalController = TestBed.inject(ModalController);`

Then
`spyOn(modalController, 'dismiss);`